exceptions.py

```diff
@@ -233,6 +233,15 @@ def __init__(self, level, line_number, unquotedtext=None):
                          line_number=line_number)
 
 
+class MisspelledAtCommand(HedyException):
+    # TODO adjust the parameters, will only put this in en/messages.po
```

Comment:
Hi @akseron!

Is this something you want to do in this PR or can we merge this now and do this later on?

In any case, can you add a date and your initials to the TODO so we know who to ask in the future? :)

Reply:
Oh I had forgotten to remove that, that was part of an earlier part of the code adjustment.
Will keep in mind to do so!

Reply:
Does that mean it can be removed? If so, please do :) 

I have fixed the po issues, so I think.... we might be ready to merge?!

Reply:
Hahaha, I did want to ask about one more thing before that. In the testing in tests/test_level/test_level_04.py I had changed the parameter for skipping faulty code, however I'm not 100% sure that I did it in the right place. If that is correct it should be ready to merge :D